src/obr/cli.py

```diff
@@ -387,38 +394,48 @@ def archive(ctx: click.Context, **kwargs):
         )
 
     # setup target folder
-    path = Path(target_folder + f"/{time}")
-    if not path.exists():
-        logging.info(f"creating {path}")
-        path.mkdir()
+    if not target_folder.exists():
+        logging.info(f"creating {str(target_folder)}")
+        target_folder.mkdir()
 
     skip = kwargs.get("skip_logs")
     if not skip:
         # iterate cases and copy log files into target repo
         for job in jobs:
+            # copy signac state point
+            signac_statepoint = Path(job.path) / "signac_statepoint.json"
+            if not signac_statepoint.exists():
+                continue
+            else:
+                target_file = (
+                    target_folder / f"workspace/{job.id}/signac_statepoint.json"
+                )
+                print("DEBUG", target_folder, signac_statepoint)
+                copy_to_archive(repo, use_github_repo, signac_statepoint, target_file)
+
             case_folder = Path(job.path) / "case"
             if not case_folder.exists():
                 logging.info(f"Job with {job.id=} has no case folder.")
                 continue
 
             # skip if either the most recent obr action failed or the label is set to "not success"
-            case = OpenFOAMCase(str(case_folder), job)
-            if not case.was_successful():
-                logging.info(
-                    f"Skipping Job with {job.id=} due to recent failure states."
-                )
-                continue
+            # case = OpenFOAMCase(str(case_folder), job)
```

Comment:
I temporarily removed this. I think it didn't work yet, it think it should be in the final version however.